<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
   <item>
    <layout class="QVBoxLayout" name="QVBoxLayout_left" stretch="2,1">
     <item>
      <widget class="QGraphicsView" name="graphicsView_left"/>
     </item>
     <item>
      <widget class="QTableViewCustom" name="tableView_left">
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="QVBoxLayout_right" stretch="2,1">
     <item>
      <widget class="QGraphicsView" name="graphicsView_right"/>
     </item>
     <item>
      <widget class="QTableViewCustom" name="tableView_right">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="QVBoxLayout_ctrl_stat" stretch="0,1">
     <item>
      <widget class="MatplotlibWidget" name="matplotlibWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>192</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_ctrl_result">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="ctrl">
        <attribute name="title">
         <string>Control</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_selimgDescription">
             <property name="text">
              <string>Selected image:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_selimg">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_imagetype">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2_bc">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Brightness:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="horizontalSlider_brightness">
             <property name="minimum">
              <number>-250</number>
             </property>
             <property name="maximum">
              <number>250</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="horizontalSlider_contrast">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Contrast:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="toolButton_brightness_reset">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="toolButton_contrast_reset">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_rotation">
           <item row="0" column="2">
            <widget class="QSpinBox" name="spinBox_rot">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string>°</string>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>360</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QToolButton" name="toolButton_rotcw">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↻</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="toolButton_rotccw">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↺</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Image pixelsize  = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_imgpxsize">
               <property name="text">
                <string>None</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_imgpxsizeUnit">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Marker radius:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_markerSize">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::WheelFocus</enum>
               </property>
               <property name="accelerated">
                <bool>false</bool>
               </property>
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Marker diameter ≙</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_markerSizeNano">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_markerSizeNanoUnit">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>um</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_selectedTableDescription">
               <property name="text">
                <string>Selected Table:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_selectedTable">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QToolButton" name="toolButton_importPoints">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>import points</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_exportPoints">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>export points</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_test">
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_correlate">
           <property name="text">
            <string>Run correlation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="results">
        <attribute name="title">
         <string>Results</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="options">
        <attribute name="title">
         <string>Options</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTableViewCustom</class>
   <extends>QTableView</extends>
   <header>QtCustom.h</header>
  </customwidget>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>QtCustom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
